### ISSUE TITLE
opal-rspec --init leads to erro cannot load such file -- opal-sprockets (LoadError)

### ISSUE THREAD

**Issue by janbiedermann** [opal/opal-rspec#95]
```
% gem install opal-rspec
% opal-rspec --init
<internal:/usr/local/Cellar/ruby/3.3.4/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require': cannot load such file -- opal-sprockets (LoadError)
        from <internal:/usr/local/Cellar/ruby/3.3.4/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
        from /usr/local/lib/ruby/gems/3.3.0/gems/opal-rspec-1.0.0/lib/opal/rspec.rb:2:in `<top (required)>'
        from <internal:/usr/local/Cellar/ruby/3.3.4/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
        from <internal:/usr/local/Cellar/ruby/3.3.4/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
        from /usr/local/lib/ruby/gems/3.3.0/gems/opal-rspec-1.0.0/lib/opal/rspec/runner.rb:2:in `<top (required)>'
        from <internal:/usr/local/Cellar/ruby/3.3.4/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
        from <internal:/usr/local/Cellar/ruby/3.3.4/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
        from /usr/local/lib/ruby/gems/3.3.0/gems/opal-rspec-1.0.0/exe/opal-rspec:6:in `<top (required)>'
        from /usr/local/lib/ruby/gems/3.3.0/bin/opal-rspec:25:in `load'
        from /usr/local/lib/ruby/gems/3.3.0/bin/opal-rspec:25:in `<main>'
```

**Comment by janbiedermann** [comment]
```
% gem info opal-rspec

*** LOCAL GEMS ***

opal-rspec (1.0.0)
    Authors: Adam Beynon, Brady Wied, Elia Schito
    Homepage: https://github.com/opal/opal-rspec
    License: MIT
    Installed at: /usr/local/lib/ruby/gems/3.3.0

    RSpec for Opal
```

**Comment by janbiedermann** [comment]
Reading Changelog that has been fixed in 1.1.0-alpha2

**Comment by janbiedermann** [comment]
Confirmed
